### ISSUE TITLE
Getting `} expectedts-styled-plugin(9999)` when using keyframes

### ISSUE THREAD

**Issue by dalvallana** [microsoft/typescript-styled-plugin#136]
I'm getting the following errors when using keyframes without interpolation:

- `} expectedts-styled-plugin(9999)`
- `at-rule or selector expectedts-styled-plugin(9999)`

```javascript
import styled, { css, keyframes } from 'styled-components';

const blink = keyframes`
  0% {
    opacity: 0;
  }

  100% {
    opacity: 1;
  }
`;

const blinkBox = styled.div`
  animation: ${css`
    ${blink} 700ms ease-out infinite alternate
  `};
`;
```

![Screen Shot 2021-05-21 at 12 04 19](https://user-images.githubusercontent.com/3943123/119120539-af75ea80-ba2c-11eb-8046-e930a51b040a.png)

![Screen Shot 2021-05-21 at 12 01 46](https://user-images.githubusercontent.com/3943123/119120643-cddbe600-ba2c-11eb-8c2d-cb59d81be6f6.png)

![Screen Shot 2021-05-21 at 12 03 38](https://user-images.githubusercontent.com/3943123/119120666-d2080380-ba2c-11eb-982e-94ca383c839f.png)

Each error appears twice. The errors disappear when I tweak it like this:

```javascript
const blink = keyframes`
  ${'0%'} {
    opacity: 0;
  }

  ${'100%'} {
    opacity: 1;
  }
`;
```
Which is ugly and shouldn't be necessary.

**Comment by lauralouiset** [comment]
Am also having this problem!

**Comment by karanparmar1** [comment]
![image](https://user-images.githubusercontent.com/39615088/120141080-a1dd0500-c190-11eb-8b66-70cae5e5d671.png)

it goes away only, when I change line 25 to this
>    ${`${originX}: ${toastSpacing};`}

**Comment by jasonwilliams** [comment]
This is my fault, I set the plugin to start adding intellisense to keyframes blocks. The issue is being dealt with here https://github.com/microsoft/typescript-styled-plugin/issues/137
